testsuite/regressiontests/ticket-683.sli

```diff
@@ -1,77 +0,0 @@
-/*
- *  ticket-683.sli
- *
- *  This file is part of NEST.
- *
- *  Copyright (C) 2004 The NEST Initiative
- *
- *  NEST is free software: you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation, either version 2 of the License, or
- *  (at your option) any later version.
- *
- *  NEST is distributed in the hope that it will be useful,
- *  but WITHOUT ANY WARRANTY; without even the implied warranty of
- *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *  GNU General Public License for more details.
- *
- *  You should have received a copy of the GNU General Public License
- *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
- *
- */
-
-/** @BeginDocumentation
-
-Name: testsuite::ticket-683 - Ensure that NEST raises error when number of threads changed too late.
-
```

Comment:
Why is this test no longer relevant?

Reply:
Because we can now set model defaults at arbitrary times instead of just before setting the resolution or the number of threads as it used to be. I could not see any bad effect when lifting this restriction.

Reply:
Do we have tests for this as well, i.e., setting defaults at arbitrary times compared to resolution changes and change of number of threads?

Reply:
I've added a (rather minimal) test in 6e843d4.